Code/Editor/MainWindow.cpp

```diff
@@ -445,11 +446,11 @@ void MainWindow::Initialize()
 {
     m_viewPaneManager->SetMainWindow(m_viewPaneHost, &m_settings, /*unused*/ QByteArray());
 
+    InitActions();
+
     RegisterStdViewClasses();
     InitCentralWidget();
 
```

Comment:
This is a subtle change and I did quite a bit of testing to make sure nothing bad happens. Basically we want to ensure all actions are registered before creating `ViewportTitleDlg` and other related window/menu types. This way when they are configured they can request the existing action to listen for changes to it (`Qt ::triggered` events). This means code in `MainWindow.cpp` does not have to be coupled to `ViewportTitleDlg`. It's not an ideal change to have to make, but it simplifies things dramatically and I believe is the right thing to do (ideally in future we can improve how actions/shotcuts and corresponding menu items are registered/created).